bgpd/bgp_ecommunity.c

```diff
@@ -727,6 +734,55 @@ char *ecommunity_ecom2str(struct ecommunity *ecom, int format, int filter)
 						      "MM:%u", seqnum);
 			} else
 				unk_ecom = 1;
+		} else if (type == ECOMMUNITY_ENCODE_TRANS_EXP) {
```

Comment:
Non-Blocking Comment and thought for the future improvements:  It might be a good idea to move to a switch statement and function calls to handle this code.

Reply:
thanks